.gitignore

```diff
@@ -5,3 +5,12 @@
 /ecs-init-*
 *.swp
 /ecs.conf
+/.rpm-done
+/.srpm-done
+/BUILD
+/RPMS
+/SOURCES
+/SRPMS
+/ecs-init.spec
+/sources.tgz
+ecs-init/version/version.go
```

Comment:
I'm strongly opposed to adding `version.go` to `.gitignore`.  I understand that there are concerns with the file changing on every commit; I think a better option would be to remove the need to generate the file and instead use `-ldflags` to inject the values at link time.